### ISSUE TITLE
Feature request: built-in loss extrapolator

### ISSUE THREAD

**Issue by Tylersuard** [pytorch/pytorch#188216]
### 🚀 The feature, motivation and pitch

According to the Neural Scaling Laws paper (Kaplan, et. al. 2020) we can make a pretty good guess at what a model's loss will be after just a few training steps. It would be nice to include this in the training printout as like a "estimated final loss" or something.
